app/models/post.rb

```diff
@@ -0,0 +1,9 @@
+class Post < ApplicationRecord
```

Comment:
- [ ] OPTIONAL
It's a good practice to use one of the Rails Active Callbacks `after_save` to call the `update_post_counter` method every time a new post has been saved to the database, and when `after_save` is called mark the `update_post_counter` method as _private_ accordingly, which should be similar to:

```ruby
 # app/models/post.rb
  after_save :update_post_counter

  private

  def update_post_counter
    author.increment!(:posts_counter)
  end
```

Reply:
well resolved! ✔️